framework/src/task_manager/task_manager_unicast.c

```diff
@@ -108,7 +108,6 @@ int task_manager_unicast(int handle, tm_unicast_msg_t *send_msg, tm_unicast_msg_
 		}
 	}
 
-	TM_FREE(((tm_unicast_internal_msg_t *)request_msg.data)->msg);
```

Comment:
Then, who frees this? Please confirm it.

Reply:
User should free this. I will update the explanation for this API.